### PR TITLE
Lesson 2 and lesson 3 is done

### PR DESCRIPTION
**Pull request by Hashlemon**
Committed with a test sample (Jest + Husky git hooks)

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/tmp/jest_rs",
```
Comment (romos21) [inline jest.config.js:14]: I think there are too many comments

**Comment by romos21** [comment]
@Hashlemon in my opinion everything is ok

**Comment by romos21** [comment]
@Hashlemon notify me when you check comments count and remove or save them if it necessary

**Comment by Hashlemon** [comment]
> @Hashlemon notify me when you check comments count and remove or save them if it necessary

@romos21 Jest config file contains only system data now.

File: jest.config.js (hunk 1 above)
Comment (Hashlemon) [reply jest.config.js]: @romos21 Cleaned

**Pull request merged**